docker_build/target_base_jammy/Dockerfile

```diff
@@ -21,13 +21,15 @@ ENV PIP_DISABLE_PIP_VERSION_CHECK=1
 ENV PIP_NO_CACHE_DIR=1
 
 RUN mkdir -p /install
-COPY install_py_310.sh /install/install_py_310.sh
+COPY install_py_312.sh /install/install_py_312.sh
 
-RUN bash /install/install_py_310.sh
-RUN curl -sS https://bootstrap.pypa.io/get-pip.py | python3.10
-RUN python3 -m pip install -U pip
+RUN bash /install/install_py_312.sh
+RUN curl -sS https://bootstrap.pypa.io/get-pip.py | python3
+RUN python3 -m pip install --upgrade pip
+RUN python3 -m pip install --upgrade pip
```

Comment:
duplication

Reply:
Faced an issue simillar to this https://github.com/pypa/pip/issues/11978#issuecomment-1513185834
Eventually found a root-cause so this duplication will be removed